scipy/stats/_morestats.py

```diff
@@ -1910,7 +1910,23 @@ def anderson(x, dist='norm'):
     .. [6] Stephens, M. A. (1979). Tests of Fit for the Logistic Distribution
            Based on the Empirical Distribution Function, Biometrika, Vol. 66,
            pp. 591-595.
-    """  # noqa
+           
+    Examples
+    --------
+    Consider random list of numbers
+    
+    
+    >>> import numpy as np
+    >>> from scipy.stats import anderson
+    >>> np.random.seed(0)
+    >>> data = np.random.normal(size=1000)
+    >>> print(anderson(data))
+    
+    AndersonResult(statistic=0.2432179174634257, critical_values=array([0.574, 0.653, 0.784, 0.914, 1.088]), 
+    significance_level=array([15. , 10. ,  5. ,  2.5,  1. ]))
+    
+    """  
+    # noqa
```

Comment:
```suggestion
    Examples
    --------
    Test the null hypothesis that a random sample was drawn from a normal
    distribution (with unspecified mean and standard deviation).

    >>> import numpy as np
    >>> from scipy.stats import anderson
    >>> rng = np.random.default_rng()
    >>> data = rng.random(size=35)
    >>> res = anderson(data)
    >>> res.statistic
    0.8398018749744764
    >>> res.critical_values
    array([0.527, 0.6  , 0.719, 0.839, 0.998])
    >>> res.significance_level
    array([15. , 10. ,  5. ,  2.5,  1. ])

    The value of the statistic (barely) exceeds the critical value associated
    with a significance level of 2.5%, so the null hypothesis may be rejected
    at a significance level of 2.5%, but not at a significance level of 1%.

    """  # noqa
```

Reply:
@mdhaber thanks for the suggestion, This code clarifies my mistake and is clear. Since this is my first PR what formatting mistake did I make? thanks!

Reply:
At the bottom of this page:

![image](https://user-images.githubusercontent.com/6570539/198446350-c08c7323-4754-48c7-a96e-a2a1ff972d01.png)

Click details

![image](https://user-images.githubusercontent.com/6570539/198446443-b95cc4ba-6181-47b5-9699-145b9d8d96df.png)

Click the link

![image](https://user-images.githubusercontent.com/6570539/198446611-30ad618a-f09a-4b4f-9a78-62ede9dc0dc8.png)

Click Lint

![image](https://user-images.githubusercontent.com/6570539/198446773-e49c9b3c-0e58-4eb3-9b85-85d12979015a.png)

Click Run Lint Checks

```
scipy/stats/_morestats.py:1913:1: W293 blank line contains whitespace
scipy/stats/_morestats.py:1917:1: W293 blank line contains whitespace
scipy/stats/_morestats.py:1918:1: W293 blank line contains whitespace
scipy/stats/_morestats.py:1924:1: W293 blank line contains whitespace
scipy/stats/_morestats.py:1925:80: E501 line too long (108 > 79 characters)
scipy/stats/_morestats.py:1925:109: W291 trailing whitespace
scipy/stats/_morestats.py:1927:1: W293 blank line contains whitespace
scipy/stats/_morestats.py:1928:8: W291 trailing whitespace
```

Reply:
To accept my suggestion, click:
![image](https://user-images.githubusercontent.com/6570539/198447240-f66e800a-19c2-4242-9692-a71d4e6e1550.png)